lib/3scale_toolbox/cli.rb

```diff
@@ -12,9 +14,139 @@ def self.load_builtin_commands
   end
 
   def self.run(args)
-    load_builtin_commands
-    ThreeScaleToolbox.load_plugins
-    # TODO handle error
-    root_command.build_command.run args
+    ErrorHandler.error_watchdog do
+      load_builtin_commands
+      ThreeScaleToolbox.load_plugins
+      root_command.build_command.run args
+    end
+  end
+
+  # Catches errors and prints nice diagnostic messages
+  class ErrorHandler
+    def self.error_watchdog(exit_on_error: true)
+      new.error_watchdog(exit_on_error: exit_on_error) { yield }
+    end
+
+    def error_watchdog(exit_on_error:)
+      # Set exit handler
+      %w[INT TERM].each do |signal|
```

Comment:
This is still in the `ErrorHandler` module. It is something that is global and being executed in a block.

By moving it into the main module I did not simply mean move the `ErrorHandler` module contents to a different file, but move this code into the other module so it is obvious it is global.

Reply:
done